kata-deploy/examples/nginx-deployment-fc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
```

Comment:
This bunch of near-identical yaml files is a good example of why we should be auto-generating the final yaml imho.

Reply:
I think we could start using something like kustomize going forward...

Reply:
I was thinking `m4` / `sed`, but if there's a funky tool to do what we want, great! ;)